network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html

```diff
@@ -29,10 +29,18 @@ <h1>{{ page.specific.headline }}</h1>
   </div>
 
   <div class="row">
-    <div class="col-md-8 offset-md-2">
+    <style>
+      .regret-stories.show-1 .youtube-regret:nth-of-type(n+{% if page.specific.batch_size %}{{ page.specific.batch_size|add:"1" }}{% else %}11{% endif %}) {
+        display: none;
```

Comment:
note: this ensures that even if there's more than 10 regrets, only 10 are visible on initial page load so that the JS "load more" does not turn a page with 30 results into a page with 10 results, which would be super janky